### DISCUSSION TITLE
How to fix : [TypeError: Cannot read property 'initContext' of null]

### DISCUSSION THREAD

**Discussion by hamdoudhakem** [mybigday/llama.rn#61] [Q&A]
So every time I run `initLlama` I keep getting this error :  `[TypeError: Cannot read property 'initContext' of null]`, from what I undertsood this either means that the file path I chose is incorrect or that the model I'm using is not supported, the problem is I literally put my model in the same folder as my script and I have written `file://bloom_3b.gguf` so I don't believe that my path is wrong. And taking the other problem into consideration I saw that the `bloom` model is supported by `llama.cpp` so there shouldn't be any problem. 

So why am I getting this error? I tried it a second time with `llama-2-7b` and I still got the same error. for some reason I can't seem to get the path right maybe? I tried putting the model in `C:/` and use the path `file:///bloom_3b.gguf`, using vs code when pressing ctrl + clicking the path it opened the model which should mean that my path is correct, what am I supposed to do?

Also I didn't do this 

> Android
> Add proguard rule if it's enabled in project (android/app/proguard-rules.pro):
> ```
> # llama.rn
> -keep class com.rnllama.** { *; }
> ```

because using Expo with the Expo Go app I don't have the `android` directory, maybe that's related?

here is my code in case someone is wondering : 
```
const ChatSessionsScreen = (
  {navigation}: {navigation : NativeStackNavigationProp<ParamListBase>}) => {

  const [model, setModel] = useState<LlamaContext | null>(null)

  // Initial a Llama context with the model (may take a while)
  useEffect(() => {
    initLlama({      
      model: 'file:///bloom_3b.gguf',
      use_mlock: true,
      n_ctx: 2048,
      n_gpu_layers: 1, // > 0: enable Metal on iOS
      // embedding: true, // use embedding
    })
    .then(model => {
      console.log("I'm Ready!")
      setModel(model)    
    })
    .catch(err => console.log(err))       
  }, [])      

  const routeParams : ChatStyleProps = {model};    
  
  return <View
    style={{flex:1, backgroundColor: COLORS.secondary, justifyContent: 'center'}}>
      <Button
        title="Click me to Navigate to Chat"
        onPress={() => navigation.navigate('Chat', routeParams)}
      />
  </View>
}
export default ChatSessionsScreen
```

**Comment by glRajkumar-haive** [comment]
I am also having same error.
